ArtifactOntology.ttl

```diff
@@ -3597,7 +3597,7 @@ cco:RadioCommunicationInterferenceArtifactFunction rdf:type owl:Class ;
 ###  http://www.ontologyrepository.com/CommonCoreOntologies/RadioCommunicationReceptionArtifactFunction
 cco:RadioCommunicationReceptionArtifactFunction rdf:type owl:Class ;
                                                 rdfs:subClassOf cco:CommunicationReceptionArtifactFunction ;
-                                                cco:definition "A Communication Reception Artifact Function that inheres in an Artifact that is capable of receiving information transmitted from another Artifact to which it is not connected by an electrical conductor."@en ;
+                                                cco:definition "A Communication Reception Artifact Function that is realized during events in which an Artifact receives information transmitted from another Artifact, where the transmission of information occurs using radio waves."@en ;
```

Comment:
Please consider the following phrasing: 
"A Communication Reception Artifact Function that, if realized, is realized during processes in which an Artifact receives information transmitted from another Artifact, where the transmission of information occurs using radio waves."